tern/formats/default/generator.py

```diff
@@ -58,6 +59,19 @@ def print_full_report(image, print_inclusive):
     return notes
 
 
+def print_layer_report(layer):
```

Comment:
In a separte PR, I wonder if we should use this function for the `-y/--layer` command line option instead of all the logic we use now?

Reply:
Yes! There is some other functionality we can consolidate to solve the issue of unrecognized Bazel commands.